sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultAccessControlClient.java

```diff
@@ -0,0 +1,180 @@
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+import com.azure.security.keyvault.administration.implementation.models.KeyVaultErrorException;
+import com.azure.security.keyvault.administration.implementation.models.RoleAssignment;
+import com.azure.security.keyvault.administration.implementation.models.RoleAssignmentProperties;
+import com.azure.security.keyvault.administration.implementation.models.RoleDefinition;
+
+/**
+ * The {@link KeyVaultAccessControlClient} provides synchronous methods to view and manage Role Based Access for the
+ * Azure Key Vault. The client supports creating, listing, updating, and deleting {@link RoleAssignment}.
+ * Additionally, the client supports listing {@link RoleDefinition}.
+ */
+@ServiceClient(builder = KeyVaultAccessControlClientBuilder.class)
+public class KeyVaultAccessControlClient {
+    private final KeyVaultAccessControlAsyncClient asyncClient;
+
+    /**
+     * Creates an {@link KeyVaultAccessControlClient} that uses a {@code pipeline} to service requests
+     *
+     * @param asyncClient The {@link KeyVaultAccessControlAsyncClient} that the client routes its request through.
+     */
+    KeyVaultAccessControlClient(KeyVaultAccessControlAsyncClient asyncClient) {
+        this.asyncClient = asyncClient;
+    }
+
+    /**
+     * Get the vault endpoint URL.
+     *
+     * @return The vault endpoint URL.
+     */
+    public String getVaultUrl() {
+        return asyncClient.getVaultUrl();
+    }
+
+    /**
+     * Get all role definitions that are applicable at scope and above.
+     *
+     * @param scope   The scope of the role definition.
+     * @param context Additional context that is passed through the HTTP pipeline during the service call.
+     * @return A {@link PagedIterable} of {@link RoleDefinition role definitions}.
+     * @throws KeyVaultErrorException if the operation is unsuccessful.
+     * @throws NullPointerException   if the {@code scope} is null.
+     */
+    public PagedIterable<RoleDefinition> listRoleDefinitions(String scope, Context context) {
+        return listRoleDefinitions(scope, null, context);
+    }
+
+    /**
+     * Get all role definitions that are applicable at scope and above.
+     *
+     * @param scope   The scope of the role definition.
+     * @param filter  The filter to apply on the operation. Use a "atScopeAndBelow" filter to search below the given
+     *                scope as well.
+     * @param context Additional context that is passed through the HTTP pipeline during the service call.
+     * @return A {@link PagedIterable} of {@link RoleDefinition role definitions}.
+     * @throws KeyVaultErrorException if the operation is unsuccessful.
+     * @throws NullPointerException   if the {@code scope} is null.
+     */
+    public PagedIterable<RoleDefinition> listRoleDefinitions(String scope, String filter, Context context) {
```

Comment:
In discussions with the service team we determined that filter is not really used and we decided to omit it from the public api for now in .NET.

Reply:
Thanks Chris, I will remove it for now.